content/docs/ui/account-and-settings/troubleshooting-delays-and-latency.md

```diff
@@ -40,27 +40,52 @@ Experiencing delays? Check our [Status Page](http://status.sendgrid.com/) for th
 
 More often that not, one of the 3 suggestions above will resolve a latency problem. However, some latency issues are simply due to the quality of your connection or traffic shaping. There is always the possibility that network problem issue lies with either yours or your ISPs configuration. Below are a number of methods that will help you determine where a latency issue really is.
 
-**Ping**
-
-1. We'll begin with the simplest method, Ping! You should always test ping to help determine response time and TTL (time to live) in milliseconds. In our troubleshooting scenario below, we'll run a ping command to our SMTP server at **smtp.sendgrid.net**
-2. How to ping using _Mac_: Open “terminal” or “network utility”. Type “ping smtp.sendgrid.net” and you will see a ping occur continually. Notice that on Mac there is no need to specify a parameter for continuous ping as in Windows.
-3. How to ping using _Windows_: Open your command prompt, usually by hitting the Windows key, typing "cmd" and hitting enter. Type “ping smtp.sendgrid.net” followed by the enter key. This will create a single ping to one of SendGrid’s IP addresses. Adding the parameter “-t” will allow for a continuous ping.
+**Installing hping**
+- Open "terminal". Type "brew install hping".

```

Comment:
We should provide instructions for our windows users as well, potentially linux too?

Reply:
I'm planning to change it to just link to hping's download docs. This will get out of scope fast with the different linux package managers and ways to set PATH